### ISSUE TITLE
Firefox 3 updated

### ISSUE THREAD

**Issue by GoogleCodeExporter** [noahocker/winetricks#96]
```
fixes multiple security bugs
```

Original issue reported on code.google.com by `t...@hush.ai` on 22 Jun 2011 at 7:53

**Comment by GoogleCodeExporter** [comment]
```
revised patch, updates firefox3 verb from 3.6.17 to 3.6.18 and firefox verb 
from 4.0.1 to 5.0

According to Asa Dotzler "the planned security update for Firefox 4
is Firefox 5." 
<url:http://groups.google.com/group/mozilla.dev.planning/browse_thread/thread/c6
bfb8eb74bc0a04/7a6cdf12dc5db986>
```

Original comment by `t...@hush.ai` on 22 Jun 2011 at 8:49

Attachments:
- [firefox.diff](https://storage.googleapis.com/google-code-attachments/winetricks/issue-96/comment-1/firefox.diff)

**Comment by GoogleCodeExporter** [comment]
```
Applied (with slight changes), thanks.  Fix released, too, as version 20110629.
```

Original comment by `daniel.r...@gmail.com` on 30 Jun 2011 at 5:23
- Changed state: **Fixed**
